tests/motor/motor_param_unittest.py

```diff
@@ -490,49 +567,73 @@ def test_time_sp_large_positive(self):
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorPositionValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorPositionSpValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorRampDownSpValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorRampUpSpValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorSpeedValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorSpeedSpValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorSpeedPValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorSpeedIValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorSpeedDValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorStateValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorStopActionValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorStopActionsValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorTimeSpValue, param=params))
-    suite.addTest(ptc.ParameterizedTestCase.parameterize(TestTachoMotorDummy, param=params))
+ev3_params = {
+    'motor': ev3.Motor('outA'),
+    'port': 'outA',
+    'driver_name': 'lego-ev3-l-motor',
+    'commands': ['run-forever', 'run-to-abs-pos', 'run-to-rel-pos', 'run-timed', 'run-direct', 'stop', 'reset'],
+    'stop_actions': ['coast', 'brake', 'hold'],
+    'supports_encoder_polarity': True,
+}
+evb_params = {
+    'motor': ev3.Motor('evb-ports:outA'),
+    'port': 'evb-ports:outA',
+    'driver_name': 'lego-ev3-l-motor',
+    'commands': ['run-forever', 'run-to-abs-pos', 'run-to-rel-pos', 'run-timed', 'run-direct', 'stop', 'reset'],
+    'stop_actions': ['coast', 'brake', 'hold'],
+    'supports_encoder_polarity': True,
+}
+brickpi_params = {
+    'motor': ev3.Motor('ttyAMA0:MA'),
+    'port': 'ttyAMA0:MA',
+    'driver_name': 'lego-nxt-motor',
+    'commands': ['run-forever', 'run-to-abs-pos', 'run-to-rel-pos', 'run-timed', 'run-direct', 'stop', 'reset'],
+    'stop_actions': ['coast', 'hold'],
+    'supports_encoder_polarity': False,
+    'speed_pid': { 'kP': 1000, 'kI': 60, 'kD': 0 },
+    'hold_pid': { 'kP': 20000, 'kI': 0, 'kD': 0 },
+}
+pistorms_params = {
+    'motor': ev3.Motor('pistorms:BAM1'),
+    'port': 'pistorms:BAM1',
+    'driver_name': 'lego-nxt-motor',
+    'commands': ['run-forever', 'run-to-abs-pos', 'run-to-rel-pos', 'run-timed', 'stop', 'reset'],
+    'stop_actions': ['coast', 'brake', 'hold'],
+    'supports_encoder_polarity': False,
+    'speed_pid': { 'kP': 1000, 'kI': 60, 'kD': 0 },
+    'hold_pid': { 'kP': 20000, 'kI': 0, 'kD': 0 },
+}
+paramsA = pistorms_params
```

Comment:
This should probably be `paramsA = ev3_params`.

Reply:
Is this still an issue?